docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -1,55 +1,75 @@
-Respondent Management to Electronic Questionnaire
--------------------------------------------------
+Census Respondent Home to Census Electronic Questionnaire
+---------------------------------------------------------
 
 When a respondent is ready to take a survey hosted on the eq system, a set of details
 need to be passed to eq to setup the survey correctly. This data is wrapped inside a json web
-token which is attached to the end of a url, digitally signed and authorised by a respondent
-management system. This creates a clean interface for any respondent management system
+token which is attached to the end of a url, digitally signed and authorised by an appropriate
+client application. This creates a clean interface for any respondent management system
 to integrate with the eq system.
 
 Schema Definition
 =================
 
-Required Fields
----------------
+Census Required Runner Fields
+------------------------------
 
 The following metadata keys are always required for the survey runner, they do not appear in individual survey metadata definitions.
 
-``tx_id``
-  see :doc:`jwt_profile`
 ``iat``
   JWT Issued At claim, see https://tools.ietf.org/html/rfc7519#section-4.1.6
 ``exp``
   JWT Expiration Time claim, see https://tools.ietf.org/html/rfc7519#section-4.1.4
-``ru_ref``
-  The responding unit reference id - with checkletter appended
-``eq_id``
-  The eQ questionnaire id
+``jti``
+   A unique identifier for the JWT
+``tx_id``
+  see :doc:`jwt_profile`
+``response_id``
+  A unique identifier for the questionnaire response
+``questionnaire_id``
+  The Census Questionnaire ID
+``case_type``
```

Comment:
If schema_type is optional, then  should these fields be optional too? Should probably be a caveat below that one or the other should be passed

Reply:
I've added a comment to reflect this

Reply:
Need to push?

Reply:
Should be there...